Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "Apache-2.0"
 readme = "README.md"
 version = "0.0.10"
 edition = "2021"
-rust-version = "1.67.0"
+rust-version = "1.81.0"
```

Comment:
In this case, can we still use this library in Rust < 1.81.0 ? 1.81.0 was just released last week, isn't this a bit aggressive?

Reply:
We typically upgrade the version only when we use a new feature that is required in the minimum version that supports that feature.

Another thing to consider is that Rust projects should usually be somewhat prepared to upgrade all the way to latest in the event of a security patch. Rust typically only releases security patches in the latest version as it has no concept of LTS releases that get security fixes backported like other ecosystems.

In this case we're using `core::error::Error` which only became available in 1.81.0. The advantage of using the core error type is that it doesn't introduce a dependence on std.

Will a dependence on 1.81.0 affect your use?

Reply:
Thank you for your detailed explanation, I have no objections to this. And I have tested it and it works well on Ledger.